src/engine/effects/engineeffectchain.cpp

```diff
@@ -128,50 +128,42 @@ bool EngineEffectChain::processEffectsRequest(const EffectsRequest& message,
     return true;
 }
 
-bool EngineEffectChain::enableForGroup(const QString& group) {
-    GroupStatus& status = getGroupStatus(group);
+bool EngineEffectChain::enableForChannel(const ChannelHandle& group) {
+    ChannelStatus& status = getChannelStatus(group);
     if (status.enable_state != EffectProcessor::ENABLED) {
         status.enable_state = EffectProcessor::ENABLING;
     }
     return true;
 }
 
-bool EngineEffectChain::disableForGroup(const QString& group) {
-    GroupStatus& status = getGroupStatus(group);
+bool EngineEffectChain::disableForChannel(const ChannelHandle& group) {
+    ChannelStatus& status = getChannelStatus(group);
     if (status.enable_state != EffectProcessor::DISABLED) {
         status.enable_state = EffectProcessor::DISABLING;
     }
     return true;
 }
 
-EngineEffectChain::GroupStatus& EngineEffectChain::getGroupStatus(const QString& group) {
-    for (QLinkedList<GroupStatus>::iterator it = m_groupStatus.begin(),
-                 end = m_groupStatus.end(); it != end; ++it) {
-        GroupStatus& status = *it;
-        if (status.group == group) {
-            return status;
-        }
-    }
-    GroupStatus status(group);
-    m_groupStatus.append(status);
-    return m_groupStatus.last();
+EngineEffectChain::ChannelStatus& EngineEffectChain::getChannelStatus(
+        const ChannelHandle& group) {
+    return m_channelStatus[group];
 }
 
-void EngineEffectChain::process(const QString& group,
+void EngineEffectChain::process(const ChannelHandle& group,
```

Comment:
> but I wonder if the term "group" is obsolete

I guess the split is, group is a control system concept (ConfigKey is a ```(group, item)``` pair). Channels in the engine (the equivalent of mixer channels on a console mixer) have a group associated with them "[Channel1]", "[Headphone]" for interacting with the control system and for uniquely identifying them within the engine code. The group is still there for interacting with the control system but ```ChannelHandle``` is now the way to uniquely identify a channel within the engine rather than its string group.